### DISCUSSION TITLE
Invitation flow for new users

### DISCUSSION THREAD

**Discussion by LawnSounds** [authelia/authelia#6111] [Ideas]
I saw the following pull request as the only mention of a possible invitation flow for new users: https://github.com/authelia/authelia/pull/2845

I would find it useful in cases where an admin may not know the users preferred email address or user-/display name. 
It may also make it easier for first time users to set their password and 2fa in general. 

I imagine the following flow:
1. Admin sends an invitation email with a unique link and token (with an expiration?)
2. The user opens the link and fills out email, display name, password and other information/required steps

Authentik has this feature which might shed some light on the request:
https://goauthentik.io/docs/flow/stages/invitation/
https://github.com/goauthentik/authentik/tree/main/authentik/stages/invitation

**Comment by LawnSounds** [comment]
Maybe related to https://github.com/authelia/authelia/discussions/5116

**Comment by james-d-elliott** [comment]
Hey thanks for taking the time to provide this idea. We have some rudimentary tooling that could expand to this purpose in the future. However we have yet to design an admin interface for features like this. I have quite a few things that I'd likely focus on before this but I'll keep it in the back of my mind.
